code/modules/clothing/spacesuits/rig/suits/station_vr.dm

```diff
@@ -153,28 +153,259 @@
+	desc = "A lightweight first responder hardsuit from the Commonwealth. Not suitable for combat use, but advanced myomer fibers can push the user to incredible speeds."
+	
+	icon = 'icons/obj/rig_modules_vr.dmi' // the item
+	default_mob_icon = 'icons/mob/rig_back_vr.dmi' // the onmob
+	icon_state = "medical_rig_bay"
+	item_state = null
+	suit_type = "medical hardsuit"
+	cell_type = /obj/item/weapon/cell/high
+
+	chest_type = /obj/item/clothing/suit/space/rig/baymed
+	helm_type = /obj/item/clothing/head/helmet/space/rig/baymed
+	boot_type = /obj/item/clothing/shoes/magboots/rig/baymed
+	glove_type = /obj/item/clothing/gloves/gauntlets/rig/baymed
+
+	// speedy paper
+	slowdown = -0.5
+	armor = list("melee" = 10, "bullet" = 5, "laser" = 10, "energy" = 5, "bomb" = 25, "bio" = 100, "rad" = 20)
+
+/obj/item/weapon/rig/baymed/equipped
+	req_access = list(access_medical)
+	
+	initial_modules = list(
+		/obj/item/rig_module/maneuvering_jets,
+		/obj/item/rig_module/sprinter,
+		/obj/item/rig_module/pat_module,
+		/obj/item/rig_module/rescue_pharm
+	)
+
+/obj/item/clothing/head/helmet/space/rig/baymed
+	icon = 'icons/inventory/head/item_vr.dmi'
+	default_worn_icon = 'icons/inventory/head/mob_vr.dmi'
+	icon_state = "medical_rig_bay"
+	item_state = null
+	sprite_sheets = ALL_VR_SPRITE_SHEETS_HEAD_MOB
+	sprite_sheets_obj = ALL_VR_SPRITE_SHEETS_HEAD_ITEM
+
+/obj/item/clothing/suit/space/rig/baymed
+	icon = 'icons/inventory/suit/item_vr.dmi'
+	default_worn_icon = 'icons/inventory/suit/mob_vr.dmi'
+	icon_state = "medical_rig_bay"
+	item_state = null
+	sprite_sheets = ALL_VR_SPRITE_SHEETS_SUIT_MOB
+	sprite_sheets_obj = ALL_VR_SPRITE_SHEETS_SUIT_ITEM
+
+/obj/item/clothing/shoes/magboots/rig/baymed
+	icon = 'icons/inventory/feet/item_vr.dmi'
+	default_worn_icon = 'icons/inventory/feet/mob_vr.dmi'
+	icon_state = "medical_rig_bay"
+	item_state = null
+	sprite_sheets = null
+	sprite_sheets_obj = null
+
+/obj/item/clothing/gloves/gauntlets/rig/baymed
+	icon = 'icons/inventory/hands/item_vr.dmi'
+	default_worn_icon = 'icons/inventory/hands/mob_vr.dmi'
+	icon_state = "medical_rig_bay"
+	item_state = null
+	sprite_sheets = null
+	sprite_sheets_obj = null
+
+// Engineering/'Industrial' rig from bay
+/obj/item/weapon/rig/bayeng
+	name = "\improper Commonwealth engineering hardsuit control module"
+	desc = "An advanced construction hardsuit from the Commonwealth. Built like a tank. Don't expect to be taking any tight corners while running."
+	
+	icon = 'icons/obj/rig_modules_vr.dmi' // the item
+	default_mob_icon = 'icons/mob/rig_back_vr.dmi' // the onmob
+	icon_state = "engineering_rig_bay"
+	item_state = null
+	suit_type = "engineering hardsuit"
+	cell_type = /obj/item/weapon/cell/super
+
+	chest_type = /obj/item/clothing/suit/space/rig/bayeng
+	helm_type = /obj/item/clothing/head/helmet/space/rig/bayeng
+	boot_type = /obj/item/clothing/shoes/magboots/rig/bayeng
+	glove_type = /obj/item/clothing/gloves/gauntlets/rig/bayeng
```

Comment:
give this mf insuls

Reply:
isnt the industrial hardsuit the one miners ~don't use~ can buy from the vendor anyway? if its for engineering purposes compare with the EVA hardsuit

Reply:
The one in the vendor is a special, nerfed version, the `/vendor` subtype.
I mostly went based on the 'real' industrial one. Though I could have gone with the CE's hardsuit I guess...

Reply:
huh. so it is.

anyways if it's intended to be used by engineering it should still probably get insuls anyway

Reply:
Ye fair, I'm away from computer but I'll do that tomorrow.